config/stack_noaa.yaml

```diff
@@ -53,25 +53,25 @@ netcdf:
   version_cxx: 4.3.1
 
 nccmp:
-  build: YES
+  build: NO
   version: 1.8.7.0
 
 nco:
   build: NO
   version: 4.9.3
 
 cdo:
-  build: YES
+  build: NO
   version: 1.9.8
 
 pio:
-  build: YES
-  version: 2.5.2
+  build: NO
+  version: 2.5.1
   enable_pnetcdf: NO
   enable_gptl: NO
 
 esmf:
-  build: YES
+  build: NO
```

Comment:
please revert to develop

Reply:
Done

Reply:
still shows `NO`

Reply:
Yes, it does.  I think I must not have pushed my changes.  Then, there was the production switch so I could no longer access my files.  Mallory Row was kind enough to grab my files and get them to me on Venus, but I haven't had a chance to get on WCOSS yet again.  I hope to do that soon.  Sorry for the confusion.